### PR TITLE
moved the sentry sdn into the secret rather than the config

### PR DESCRIPTION
**Pull request by goern**

### CODE REVIEW

**Comment by goern** [comment]
recheck

File: playbooks/create_configmap_and_secret.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  tags:
+    - openshift
+    - thoth
+
+  hosts: localhost
+  gather_facts: false
+  connection: local
+
+  roles:
+    - role: thoth-station.configmaps_and_secrets
```
Comment (fridex) [inline playbooks/create_configmap_and_secret.yaml]: So we will track changes in the `configmaps-and-secrets` repo? How will they be updated?

Comment (goern) [reply playbooks/create_configmap_and_secret.yaml]: ja, that role will deploy the configmap/secret template, the templates themselves are created via core_templates role, and the template files themselves still life in each repository.

Comment (fridex) [reply playbooks/create_configmap_and_secret.yaml]: How do we update them on galaxy?

Comment (goern) [reply playbooks/create_configmap_and_secret.yaml]: the roles? they are repositories to be importet into ansible-galaxy... `ansible-galaxy import thoth-station core-imagestreams`

**Comment by goern** [comment]
recheck

**Pull request merged**